config/env/production.js

```diff
@@ -139,11 +139,13 @@ module.exports.userEnvVar = {
 };
 
 const uaaCredentials = appEnv.getServiceCreds('uaa-client');
+const uaaOptions = {
+  clientID: uaaCredentials.client_id,
+  clientSecret: uaaCredentials.client_secret,
+};
 module.exports.passport = {
   uaa: {
-    options: {
-      clientID: uaaCredentials.client_id,
-      clientSecret: uaaCredentials.client_secret,
-    },
+    options: uaaOptions,
+    adminOptions: uaaOptions,
```

Comment:
options and adminOptions are temporarily the same?

Reply:
yes, the credentials are the same since we;re just using a single instance of the uaa service. The callback urls are different but those are configured in `./config/passport.js`